src/core/ButtonIcon/style.ts

```diff
@@ -172,6 +172,11 @@ export const StyledButtonIcon = styled(IconButton, {
 })`
   padding: 0;
 
+  :focus {
+    outline: 5px auto Highlight;
```

Comment:
ooh `highlight` is convenient 💡 How is the color value calculated? Is it based on whatever the browser defines?

Reply:
Yes, exactly. `Highlight` is the default value for focus colors in browsers like Firefox and Safari. Chrome, on the other hand, has specified its own value as `-webkit-focus-ring-color`, which is different in color but achieves the same thing.